dcrpm/rpmutil.py

```diff
@@ -202,7 +202,8 @@ def check_rpmdb_indexes(self):
                     lambda proc: proc.returncode != StatusCode.SEGFAULT,
                     lambda proc: len(proc.stdout.splitlines()) >= 1,
                     lambda proc: any(
-                        line.startswith("rpm-") for line in proc.stdout.splitlines()
+                        line.startswith("rpm-") or line.startswith("yum-")
```

Comment:
Whatever case is relying on this will be broken with dnf, but I guess this doesn't make it worse.

Reply:
fwiw all systems I've seen with DNF have rpm-libs show up in here, so they pass the check as they should